src/main/java/com/example/decafe/HelloController.java

```diff
@@ -174,31 +168,19 @@ public void switchToGameScreen() throws IOException { // if button PLAY AGAIN is
 
     // jump to instructions
     public void switchToInstructions() throws IOException {
```

Comment:
Did you rename the fxml file from Instructions.fxml to instructions.fxml ?

Reply:
yes, does it cause any problems on your side? 🤔

Reply:
No, just becasue I don't see any change on the Instructions.fxml file !

Reply:
@albiosz is this resolved?

Reply:
![grafik](https://github.com/user-attachments/assets/764264db-e3c9-430a-bc28-e749926db881)
In my case the file is actually lower case 🤔

Is it Instructions.fxml or instructions.fxml at your machines?

Reply:
Yes, seems to be lower case.

Reply:
So everything is fine, is there still a problem on your machines? @julien-jungo @Mu2menM

Reply:
It's fine